test/modules/ModuleLib/NetworkController/NetworkControllerConstants.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.NetworkController
+{
+    public class NetworkControllerConstants
```

Comment:
once move inside TestConstants class, rename it.
```suggestion
    public static class NetworkController
```